pennylane/transforms/metric_tensor.py

```diff
@@ -37,6 +37,16 @@ def expand_fn(tape, approx=None, allow_nonunitary=True, aux_wire=None, device_wi
 def metric_tensor(tape, approx=None, allow_nonunitary=True, aux_wire=None, device_wires=None):
     r"""Returns a function that computes the metric tensor of a given QNode or quantum tape.
 
+    The metric tensor convention we employ here has the following form:
+
+    .. math::
+
+        \text{metric_tensor}_{i, j} = \text{Re}\left[ \langle \partial_i \psi(\bm{\theta}) | \partial_j \psi(\bm{\theta}) \rangle
+        - \langle \partial_i \psi(\bm{\theta}) | \psi(\bm{\theta}) \rangle \langle \psi(\bm{\theta}) | \partial_j \psi(\bm{\theta}) \rangle \right]
+
+    with short notation :math:`| \partial_j \psi(\bm{\theta}) \rangle := \frac{\partial}{\partial \theta_j}| \psi(\bm{\theta}) \rangle`.
+    It is closely related to the quantum fisher information matrix, see :func:`~.qinfo.quantum_fisher` and eq. (26) in `arxiv:2103.15191 <https://arxiv.org/abs/2103.15191>`_.

```

Comment:
Not sure whether you mean (27) here, which is the one mentioned in the `qinfo.quantum_fisher` function? :)
```suggestion
    It is closely related to the quantum fisher information matrix, see :func:`~.qinfo.quantum_fisher` and eq. (27) in `arxiv:2103.15191 <https://arxiv.org/abs/2103.15191>`_.

```

Reply:
My understanding from our chat on slack was that `metric_tensor` implements eq. (26) / (27) without _any_ prefactor, such that `qinfo.quantum_fisher`, which is supposed to be eq. (27), simply does `4 * metric_tensor`.

In case `metric_tensor` computes $2 \text{Re}..$ I would have to change `quantum_fisher` to `2 * metric_tensor`.

Reply:
Ah, wait. I think I was unclear with my comment: It sounds like the "see :func:..." in the docstring refers to the QFI. But it refers to the metric tensor, right? (26) is neither of the two, having the halved prefactor of 2, right? :sweat_smile:

Reply:
Ah right, now I get it! Yes, makes more sense to refer to eq. (27)!